portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,35 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    private ArrayList<String> comments;
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Gia-Huy Gonzalez!</h1>");
+    comments = new ArrayList<>();
+    comments.add("Make in your face like a samsung");
+    comments.add("I took it and I ran for it");
+    comments.add("Smile pearl white, shine so bright");
+    
+    String test = "Hello Gia-Huy Gonzalez!";
+    String json = convertToJson(comments);
+
+    //response.setContentType("text/html;");
```

Comment:
It's a good idea to make sure that your code doesn't have any extraneous debugging comments left in. In this case, these commented out "response" calls can be safely removed.